app/graphql/types/user_type.rb

```diff
@@ -14,6 +14,14 @@ def authorized?(_object, _args, context)
         context[:current_school_admin].present?
       end
     end
+    field :current_standing_name, String, null: true
+
+    def current_standing_name
+      if Schools::Configuration.new(object.school).standing_enabled?
+        object.user_standings.where(archived_at: nil).last&.standing&.name ||
+          Standing.where(school: object.school, default: true).first&.name
+      end
+    end
```

Comment:
This has to be batched!

Reply:
https://github.com/pupilfirst/pupilfirst/pull/1468/commits/9eeb9b8b648523b33827947472f333c2ab45c001